docs/features.md

```diff
@@ -0,0 +1,48 @@
+---
+Order: 20
+Title: Features
+Description: Features of the Cake.Issues.PullRequests.Tfs addin.
+---
+The [Cake.Issues.PullRequests.Tfs addin] provides the following features.
+
+:::{.alert .alert-info}
+There's a [demo repository] available which you can fork and to which you can create pull requests to test the integration functionality.
+:::
+
+# Basic features
+
+* Writes issues as comments to Team Foundation Server (TFS) or [Azure DevOps] pull requests.
+* Identification of pull requests through source branch or pull request ID.
+* Comments written by the addin will be rendered with a specific icon corresponding to the state of the issue.
+* Adds rule number and, if provided by the issue provider, link to the rule description to the comment.
+* Support for issues messages formatted in Markdown format.
+
+# Supported capabilities
```

Comment:
[MD025](https://github.com/DavidAnson/markdownlint/blob/master/doc/Rules.md#md025---multiple-top-level-headings-in-the-same-document
) multiple top level headings in the same document.